### PR TITLE
Add a missing va_start() and a missing va_end()

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Like
JIRA issue: [CORE-14193](https://jira.reactos.org/browse/CORE-14193)

## Proposed changes

- [D3D9] Add a missing va_end()
- [ARPING] Add a missing va_start(), Move 2 va_end()

### CODE REVIEW

File: modules/rosapps/applications/cmdutils/arping/arping.c [hunk 1]
```diff
@@ -107,6 +107,8 @@ void FormatOutput(UINT uID, ...)
 
     if(pBuf != Buf)
         LocalFree(pBuf);
+
+    va_end(valist);
```
Comment (ThFabba) [inline modules/rosapps/applications/cmdutils/arping/arping.c]: This actually doesn't fix this function's problem. You can't use a va_list twice as is done here.
Can you change it so it calls va_end after the first function call, then does another va_start/va_end pair around the second? Thanks.

Comment (SergeGautherie) [reply modules/rosapps/applications/cmdutils/arping/arping.c]: I'll do that.
This last `va_end()` could be done much sooner too, couldn't it? Or must `valist` remain active until then?

Comment (ThFabba) [reply modules/rosapps/applications/cmdutils/arping/arping.c]: Correct, you can do va_start(); function(); va_end(); both times.

Comment (SergeGautherie) [reply modules/rosapps/applications/cmdutils/arping/arping.c]: Done.

**Comment by SergeGautherie** [comment]
> FormatOutput in arping was a blind copy paste from another tool

It was copied from "ping". That initial code has been replaced in the meantime. I'll file an issue to review/port that to "arping".

**Pull request merged**